job_offers_data_load/update_MongoDB_procedure.py

```diff
@@ -0,0 +1,48 @@
+import pandas
+from job_offers_data_load import MongoDB_connection_functions
+import datetime
+import pymongo
+
+
+def check_update_or_create_MongoDB(new_data: pandas.DataFrame,
```

Comment:
Are you checking mongodb for upgrades or are you willing to recreate mongodb project?